packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1073,43 +1083,52 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
           // This is a fatal error.
           didFatal = true;
           onUncaughtError(thrownValue);
-          break;
```

Comment:
Flow wasn't sure `nextUnitOfWork` isn't null without an explicit `if` / `else` here. 
There's another `break` below the whole thing.